java/nouveau/pom.xml

```diff
@@ -0,0 +1,216 @@
+<!--
+    Copyright 2022 Cloudant. All rights reserved.
```

Comment:
stray copyright notice

Reply:
there are others, too, just leaving this as a placeholder. We probably also don't need a dedicated LICENSE copy

Reply:
ah oof, yes, I copied from clouseau's pom.xml, my bad